eng/helix/Helix.proj

```diff
@@ -58,38 +58,38 @@
   <!-- Linux gnu libc queues -->
   <ItemGroup Condition="$([MSBuild]::IsOSPlatform(Linux)) and !$(PackageRid.Contains(musl))">
     <!-- linux-arm64 -->
-    <HelixAvailableTargetQueue Include="ubuntu.1804.armarch$(QueueSuffix)@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-helix-arm64v8"
+    <HelixAvailableTargetQueue Include="ubuntu.2204.armarch$(QueueSuffix)@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-12-helix-arm64v8"
                                Condition="'$(HelixArchitecture)' == 'arm64'">
-      <TestRunName>Debian 11 $(HelixArchitecture) $(HelixConfiguration)</TestRunName>
+      <TestRunName>Debian 12 $(HelixArchitecture) $(HelixConfiguration)</TestRunName>
     </HelixAvailableTargetQueue>
-    <HelixAvailableTargetQueue Include="ubuntu.1804.armarch$(QueueSuffix)@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8"
+    <HelixAvailableTargetQueue Include="ubuntu.2204.armarch$(QueueSuffix)@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04-helix-arm64v8"
```

Comment:
@richlander, there is no `ubuntu-20.04-helix-arm64v8` tag for `mcr.microsoft.com/dotnet-buildtools/prereqs`. See https://mcr.microsoft.com/v2/dotnet-buildtools/prereqs/tags/list. In fact, there is no helix image for Ubuntu 20.04 on arm64. This is a little surprising given that .NET 8 is supported on 20.04 and above: https://github.com/dotnet/core/blob/main/release-notes/8.0/supported-os.md#linux.

Reply:
```suggestion
    <HelixAvailableTargetQueue Include="ubuntu.2204.armarch$(QueueSuffix)@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-helix-arm64v8"
```

Let's just use 22.04 for Arm64 then. We could create 20.04 image, but it probably doesn't matter. We're covering the full combination of 20.04 and Arm64 and x64 (and get 22.04 for free). Seems fine to me.